backend/tests/clubs/test_ticketing.py

```diff
@@ -794,12 +813,13 @@ def test_get_cart_replacement_required_sold_out(self):
         to_add = set(map(lambda t: str(t.id), tickets_to_add))
         self.assertEqual(len(in_cart & to_add), 0, in_cart | to_add)
 
-    def test_initiate_checkout(self):
+    def test_initiate_checkout_free_and_non_free_tickets(self):
```

Comment:
nit: would prefer to create a separate test case for this.

Reply:
Since there are only two routes to purchasing a ticket: `cart_total=0` and `cart_total>0`, including free tickets here doesn't make the route here special since it's the case `cart_total>0`. Would you rather have me revert to the initial name?

Reply:
Let's split this into 3 tests -- the original (tests only paid tickets), test only free tickets, and test a mix.

Reply:
will do